src/todoItem/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
```

Comment:
Please rename the folder from `todoItem` to `TodoItem` so that its name shows it's a component